src/core/worker.ts

```diff
@@ -191,9 +191,7 @@ export class Worker extends EventEmitter {
   }
 
   private async poll(nQueue = 0) {
-    if (!this.running) {
-      return;
-    }
+    if (!this.running) return;
```

Comment:
my personal preference is never to do such single line code "optimizations":
1. ...
2. using consistent style
2. inexperienced developer may be adding extra code that they think is grouped by condition but really is just standalone statement

```diff
-    if (!this.running) return;
+    if (!this.running) console.log("running"); return;
```